components/theme/app/AsideBottom.vue

```diff
@@ -1,13 +1,46 @@
 <template>
-  <div v-if="$docus.currentPath.value.startsWith('/docs')">
-    <AppLink v-if="lastRelease" to="/releases" class="flex items-center group nuxt-text-highlight-hover mt-4">
-      <IconNuxt class="w-5 h-5 mr-2" />
-      <span>{{ $t('common.version') }}: {{ lastRelease }}</span>
-    </AppLink>
-    <AppLink href="https://v3.nuxtjs.org" class="flex items-center group nuxt-text-highlight-hover mt-4">
-      <IconNuxt class="w-5 h-5 mr-2 text-primary" />
-      <span>{{ $t('common.version') }}: v3.x (Beta)</span>
-    </AppLink>
+  <div>
+    <a
+      v-if="
+        masterCoursesLink &&
+        ($docus.currentPath.value.startsWith('/docs') || $docus.currentPath.value.startsWith('/examples'))
+      "
+      :href="masterCoursesLink.href"
+      rel="noopener nofollow noreferrer"
+      target="_blank"
```

Comment:
I would recommand using `AppLink v-if="..." :href="...">` which handles `blank` and `rel`.
I also recommand `rel="noopener"` since this is a sponsored link.

Reply:
Thanks, @smarroufin, I replaced the `a` with an `AppLink`.